packages/salesforcedx-vscode-apex-replay-debugger/test/vscode-integration/commands/quickLaunch.test.ts

```diff
@@ -112,6 +114,7 @@ describe('Quick launch apex tests', () => {
       undefined,
       'MyClass'
     ]);
+    expect(writeResultFilesStub.called).to.equal(true);
```

Comment:
Should we be testing the arguments of this call? At least the TestResult value?

Reply:
Added, thanks Jag!